src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/Intellisense.cs

```diff
@@ -264,9 +264,19 @@ private IIntellisenseResult Finalize(IIntellisenseContext context, IntellisenseD
                 handler.Run(context, intellisenseData, resultSuggestions);
             }
 
-            intellisenseData.Suggestions.Sort(culture);
-            intellisenseData.SubstringSuggestions.Sort(culture);
-            resultSuggestions.Sort(new IntellisenseSuggestionComparer(culture));
+            // InvariantCulture
+            if (culture?.LCID == 127)
```

Comment:
>  if (culture?.LCID == 127) [](http://example.com/codeflow?start=10&length=27)

could we just set culture= null and share 276..278 (rather than copy code)